src/sonic-config-engine/config_samples.py

```diff
@@ -194,13 +195,46 @@ def generate_l2_config(data):
             data['VLAN_MEMBER']['Vlan1000|{}'.format(port)] = {'tagging_mode': 'untagged'}
     return data
 
+def generate_appliance_config(data):
+    data['DEVICE_METADATA']['localhost']['hostname'] = 'sonic'
+    data['DEVICE_METADATA']['localhost']['switch_type'] = 'dpu'
+    data['DEVICE_METADATA']['localhost']['type'] = 'SonicHost'
```

Comment:
this is SonicDpu

Reply:
I removed this file from the PR. I'll raise another PR after we close the question with the default topology name for the DPU.